Source/WebInspectorUI/UserInterface/Views/LayoutDetailsSidebarPanel.js

```diff
@@ -171,17 +171,13 @@ WI.LayoutDetailsSidebarPanel = class LayoutDetailsSidebarPanel extends WI.DOMDet
 
         for (let node of WI.domManager.attachedNodes({filter: (node) => node.layoutContextType})) {
             switch (node.layoutContextType) {
-            case WI.DOMNode.LayoutContextType.Grid:
+            case WI.DOMNode.LayoutFlag.Grid:
                 this._gridNodeSet.add(node);
                 break;
 
-            case WI.DOMNode.LayoutContextType.Flex:
+            case WI.DOMNode.LayoutFlag.Flex:
                 this._flexNodeSet.add(node);
                 break;
-
-            default:
-                console.assert(false, "Unknown layout context type.", node, node.type);
-                break;
```

Comment:
I guess this isn't reachable currently, but I'd suggest leaving it so if we add another valid layout context at some point we don't just silently do nothing here.

Reply:
oh lol oops i think this was removed in a previous design and i forgot to add it back